### ISSUE TITLE
Mapping the TEL=mobile subtype

### ISSUE THREAD

**Issue by ScottPeterJohnson** [mstilkerich/rcmcarddav#390]
[VCard v3 defines the following subtypes:](https://www.rfc-editor.org/rfc/rfc2426#section-3.3.1)
```
home
msg
work
pref
voice
fax
cell
video
pager
bbs
modem
car
isdn
pcs
```
When creating contacts in Roundcube however, the following subtypes seem to pass directly to VCard format:
```
'home','work','home2','work2','mobile','main','homefax','workfax','car','pager','video',
                'assistant','other'
```
Some of these are in the spec and will be generally understood by other software, others will not. The big one is probably "mobile", which could just map to "cell". 
But "homefax" and "workfax" could be split into e.g. TYPE=home;TYPE=fax. 
(Note that Vcard V4 has its own list of subtypes.)

I'm willing to try to write a PR for this, if you agree with my diagnosis of the problem.

**Comment by mstilkerich** [comment]
Hello, thanks for digging this up, I'm quite surprised that no one ever complained about it :-). What rcmcarddav currently offers for phone numbers is the same selection as roundcube provides. In addition, it will offer custom labels (X-ABLABEL extension) if received during import of vCards. I absolutely agree we should use the standardised type values here. Looking at IANA, the current list for TEL-TYPE is: work, home, text, voice, fax, cell, video, pager, textphone, main-number. I guess the extra types in VCard3 are not needed nowadays anymore.

However when changing the list, there may be some extra work to do that will require some digging. At the least, roundcube localises the labels displayed in the addressbook, so when using a different selection here than roundcube offers a localisation of these will also be needed.

**Comment by mstilkerich** [comment]
There is a fix on master now, I will still need to update and extend the tests

**Comment by mstilkerich** [comment]
Fix is now released with v4.4.3

**Comment by ScottPeterJohnson** [comment]
Great work on fixing this so fast!
